api/src/lib/passwordEncryption.ts

```diff
@@ -0,0 +1,12 @@
+import bcrypt from 'bcryptjs';
+
+export const getPasswordHash = async (password: string): Promise<string> => {
+  return bcrypt.hashSync(password, 10);
+};
+
+export const isValidPassword = async (
+  password: string,
+  passwordHash: string,
+): Promise<boolean> => {
+  return bcrypt.compareSync(password, passwordHash);
```

Comment:
I'd prefer if you'd invoke the async method if possible

Reply:
Ok, I'll commit the change